presto-product-tests/conf/docker/singlenode-hdfs-impersonation/docker-compose.yml

```diff
@@ -4,3 +4,8 @@ services:
   presto-master:
     volumes:
       - ../../../conf/presto/etc/environment-specific-catalogs/singlenode-hdfs-impersonation/hive.properties:/docker/volumes/conf/presto/etc/catalog/hive.properties
+
+  application-runner:
```

Comment:
Why this is special case to be tested without stats?
What about tests for accessing kerberized clusters?

Reply:
Sorry about that, but please consider this PR as WIP. This one failed without this. I have to check others as well as the ones in presto-checks.